common/src/main/java/io/homeassistant/companion/android/common/data/websocket/impl/WebSocketRepositoryImpl.kt

```diff
@@ -179,6 +180,17 @@ class WebSocketRepositoryImpl @Inject constructor(
         }
     }
 
+    override suspend fun getConversation(speech: String): ConversationResponse? {
+        val socketResponse = sendMessage(
+            mapOf(
```

Comment:
For future, do we have ability to send the language of the speech to text / watch. This will be helpful in case the watch language doesn't match the language configured in HA

Reply:
Looks like it should be possible using another API once we get the text.

https://developers.google.com/ml-kit/language/identification/android

Reply:
I would think that whatever language the app is using would be enough?  I don't think we need the ML stuff for that.

Reply:
Yeah that should definitely be enough.

Reply:
oh I was thinking we needed to detect the language spoken which may not match what language the watch is in?

Reply:
we need to just get the language that the text is in. Which is based on what the users keyboard is set to I guess. But we can assume 95% certain that it matches the app language

Reply:
> oh I was thinking we needed to detect the language spoken which may not match what language the watch is in?

The docs [suggest](https://developer.android.com/reference/android/speech/RecognizerIntent#EXTRA_LANGUAGE) that it will always recognize speech based on the language returned by `Locale.getDefault()`.

Reply:
Thanks all :) I will add a `TODO` note for this so we can take care of this in future